spec/integration/change_stream_examples_spec.rb

```diff
@@ -174,10 +174,12 @@
 
         # Start Changestream Example 4
 
-        pipeline = [ {'$match' => { '$or' => [{ 'fullDocument.username' => 'alice' },
-                                              { 'operationType' => 'delete' }] } }]
-        cursor = inventory.watch(pipeline).to_enum
-        cursor.next
+        pipeline = [
+          { "$match" => { 'fullDocument.username' => 'alice' } },
+          { "$addFields" => { 'newField' => 'this is an added field!' } }
+        ];
+        changeStream = inventory.watch(pipeline).to_enum
```

Comment:
Let's use `cursor` for consistency with other examples.

Reply:
Done!